models/staging/graph/stg_columns.sql

```diff
@@ -0,0 +1,19 @@
+{{
+    config(
+        materialized='table',
+    )
+}}
+
+{% if execute %}
```

Comment:
i think techincally speaking, this one doesn't need to be a table! that check is important for the tables where we're abusing the `post_hook` feature to run some insert statements, which can't be done on views!